{
  "url": "https://api.github.com/repos/shap/shap/issues/2676",
  "repository_url": "https://api.github.com/repos/shap/shap",
  "labels_url": "https://api.github.com/repos/shap/shap/issues/2676/labels{/name}",
  "comments_url": "https://api.github.com/repos/shap/shap/issues/2676/comments",
  "events_url": "https://api.github.com/repos/shap/shap/issues/2676/events",
  "html_url": "https://github.com/shap/shap/issues/2676",
  "id": 1367923525,
  "node_id": "I_kwDOBHDcK85RiNtF",
  "number": 2676,
  "title": "XGBRegressor no longer supported by TreeExplainer ? ",
  "user": {
    "login": "iTimo31",
    "id": 71881966,
    "node_id": "MDQ6VXNlcjcxODgxOTY2",
    "avatar_url": "https://avatars.githubusercontent.com/u/71881966?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/iTimo31",
    "html_url": "https://github.com/iTimo31",
    "followers_url": "https://api.github.com/users/iTimo31/followers",
    "following_url": "https://api.github.com/users/iTimo31/following{/other_user}",
    "gists_url": "https://api.github.com/users/iTimo31/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/iTimo31/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/iTimo31/subscriptions",
    "organizations_url": "https://api.github.com/users/iTimo31/orgs",
    "repos_url": "https://api.github.com/users/iTimo31/repos",
    "events_url": "https://api.github.com/users/iTimo31/events{/privacy}",
    "received_events_url": "https://api.github.com/users/iTimo31/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 0,
  "created_at": "2022-09-09T14:35:44Z",
  "updated_at": "2022-09-09T15:11:30Z",
  "closed_at": "2022-09-09T15:11:30Z",
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "Hello, \r\nI am writing a program that first checks if TreeSHAP is compatible on the model in parameter and applies another algorithm if it is not.\r\n\r\nUntil yesterday, my XGBRegressor model was compatible, and today, this code returns False and I can't explain why :\r\n\r\n```\r\nfrom shap import explainers\r\nfrom xgboost import XGBRegressor\r\n\r\nmodel = XGBRegressor()\r\n\r\nprint(explainers.Tree.supports_model_with_masker(model, None))\r\n```\r\n\r\nThanks !",
  "closed_by": {
    "login": "iTimo31",
    "id": 71881966,
    "node_id": "MDQ6VXNlcjcxODgxOTY2",
    "avatar_url": "https://avatars.githubusercontent.com/u/71881966?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/iTimo31",
    "html_url": "https://github.com/iTimo31",
    "followers_url": "https://api.github.com/users/iTimo31/followers",
    "following_url": "https://api.github.com/users/iTimo31/following{/other_user}",
    "gists_url": "https://api.github.com/users/iTimo31/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/iTimo31/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/iTimo31/subscriptions",
    "organizations_url": "https://api.github.com/users/iTimo31/orgs",
    "repos_url": "https://api.github.com/users/iTimo31/repos",
    "events_url": "https://api.github.com/users/iTimo31/events{/privacy}",
    "received_events_url": "https://api.github.com/users/iTimo31/received_events",
    "type": "User",
    "site_admin": false
  },
  "reactions": {
    "url": "https://api.github.com/repos/shap/shap/issues/2676/reactions",
    "total_count": 0,
    "+1": 0,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/shap/shap/issues/2676/timeline",
  "performed_via_github_app": null,
  "state_reason": "completed"
}
